db/seeds.rb

```diff
@@ -6,24 +6,24 @@
+item2 = brewing.items.create!(title: 'Columbian Dark Coffee', description: 'Creamy and smooth-bodied with a smoky aroma and well-balanced flavor', price: 1500, image: File.new("#{Rails.root}/app/assets/images/coffee.bag.jpg"))
 
-item3 = merchandise.items.create!(title: 'Clear Kit', description: 'Full Kit.', price: 8000, photo: 'merch.jpg')
-item4 = merchandise.items.create!(title: 'Coffee Grinder', description: 'Classic hand crank coffee grinder', price: 2000, photo: 'grinder.jpg')
-item5 = merchandise.items.create!(title: 'Espresso Machine', description: 'Bring out the indulgent European in you', price: 12000, photo: 'espresso-machine.jpg')
-item6 = merchandise.items.create!(title: 'Coffee Maker', description: 'Chrome coffee maker, for the baller in you', price: 5000, photo: 'coffee-maker.jpg')
+item3 = merchandise.items.create!(title: 'Clear Kit', description: 'Full Kit.', price: 8000, image: File.new("#{Rails.root}/app/assets/images/merch.jpg"))
+item4 = merchandise.items.create!(title: 'Coffee Grinder', description: 'Classic hand crank coffee grinder', price: 2000, image: File.new("#{Rails.root}/app/assets/images/grinder.jpg"))
+item5 = merchandise.items.create!(title: 'Espresso Machine', description: 'Bring out the indulgent European in you', price: 12000, image: File.new("#{Rails.root}/app/assets/images/espresso-machine.jpg"))
+item6 = merchandise.items.create!(title: 'Coffee Maker', description: 'Chrome coffee maker, for the baller in you', price: 5000, image: File.new("#{Rails.root}/app/assets/images/coffee-maker.jpg"))
 
-item7 = chow.items.create!(title: 'Artichoke Souffle', description: 'Try this rich-tasting cheese, artichoke and spinach soufflé recipe for your next brunch.', price: 700, photo: 'breakfast.jpg')
-item8 = chow.items.create!(title: 'Berry Tart', description: 'Lightly sweetened berries top vanilla bean-flecked pastry cream contained in a graham cracker crust.', price: 500, photo: 'tarts.jpg')
-item9 = chow.items.create!(title: 'Caprese Panini', description: 'The classic combination of mozzarella, tomatoes and basil is known as caprese.', price: 700, photo: 'panini.jpg')
-item10 = chow.items.create!(title: 'Blueberry Muffin', description: 'blueberry muffins balance a moist, buttery crumb topping and are equally delicious with blueberries in season.', price: 400, photo: 'muffin.jpg')
-item11 = chow.items.create!(title: 'Wedge Salad', description: 'Hearts of romaine lettuce, painted with a herb vinaigrette, and grilled.', price: 800, photo: 'salad.jpg')
-item12 = chow.items.create!(title: 'Tomato Soup and Toasted Ravioli', description: 'Creamy tomatoe soup made from freash local ingrediants, garnished with our in house toasted ravioli pasta.', price: 800, photo: 'soup.jpg')
+item7 = chow.items.create!(title: 'Artichoke Souffle', description: 'Try this rich-tasting cheese, artichoke and spinach soufflé recipe for your next brunch.', price: 700, image: File.new("#{Rails.root}/app/assets/images/breakfast.jpg"))
```

Comment:
Useless assignment to variable - `item7`.<br>Line is too long. [237/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.